ffs/ffs_terms.md

```diff
@@ -0,0 +1,172 @@
+# Forum Funding System v1.0
+
+This document is written in order to clearly outline the expectations of a Forum Funding System (hereafter 'FFS') proposal, the individual or team (hereafter 'Contractor') that will fulfill it, and people that voluntarily contribute to a proposal (hereafter 'Donor') with their Monero (hereafter 'XMR').
+
+The community of Monero is really a very welcoming and friendly place. While these terms may seem strict to some, we just want to ensure that we have an environment that is fair, encourages innovation, protects XMR of contributors, and protects contractors. In other words, we want a smooth FFS experience for everyone, contractors and contributors alike.
+
+We all want Monero to succeed, and we've got an awesome community that can make it happen. Let's just treat each other well, be honest with each other about anything unexpected that may arise, and make reasonable arguments for or against something, and, you know, be adults. In doing so, we shouldn't have any issues.
```

Comment:
While pleasant, for me these last 2 paragraphs seem like a bit too much fluff. This should be a straight-forward not-TL;DR document.

Section 1 is great and does what these 2 paragraphs do - but more effectively, imho.